### PR TITLE
Setup server express

### PR DESCRIPTION
**Pull request by lee-philip-n**
Trello implementation: 
https://trello.com/c/7BSKbKHJ
https://trello.com/c/BlICZyZA

just simple set up of server connection and checking if properly serving the files.
i'm not sure if there is a need for unit test on this part? please let me know if so?

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.use(express.static(path.resolve(__dirname, '../client/dist')));
+
+app.use('/movies', routes);
+
+app.listen(3000, () => console.log('Example app listening on port 3000!'));
```
Comment (arjun0419) [inline server/server.js]: I would rewrite this console.log if you would like to keep it.

File: server/routes.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/ID:/actors', (req, res) => {
```
Comment (arjun0419) [inline server/routes.js]: What does the colon in the file path do?

Comment (lee-philip-n) [reply server/routes.js]: doesn't do anything, its just a place holder for now until i get axios feature working. I'm a bit confused on how are we going to do a get request without a search bar...

File: server/server.js (hunk 1 above)
Comment (lee-philip-n) [reply server/server.js]: ill just remove it

File: server/routes.js (hunk 2 above)
Comment (arjun0419) [reply server/routes.js]: ah i see. Thats fine for now then.

**Pull request merged**